conbench/outlier.py

```diff
@@ -0,0 +1,101 @@
+import logging
+
+import numpy as np
+import pandas as pd
+
+log = logging.getLogger(__name__)
+
+
+def remove_outliers_by_iqrdist(
```

Comment:
It would be nice to make a little library here of interchangeable  outlier detection methods including this and the one used in the plots, though we'd need consistent input and output data structures, and to separate concerns from change detection/segmentation, which probably has to happen first. But we should keep pushing down the path of modularity as we can.

Reply:
Yes, thought similar thoughts!